pwa/src/components/deleteModal/deletemodal.css

```diff
@@ -0,0 +1,3 @@
+.cancel-btn-right {
+  float: right;
+}
```

Comment:
Two things here: 

1. I would recommend working with `flex`, if you're unsure how this works [check out this guide](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) and
2. Your `DeleteModal` is wrapped in a `.DeleteModal` className, we should use this to style everything within this component, like so:

```
.DeleteModal .DeleteModal-button--cancel {
  ... 
}
```

This way, the `css` logic written in this file, can never unintentionally target elements outside of this component.